examples/opengl_boxes.rs

```diff
@@ -259,7 +262,8 @@ fn main() {
         if let Some(state) = gl_canvas_ref.draw_state().downcast_ref::<GLCanvasState>() {
             let old_size = target_size.get();
             let new_size = state.measure();
-            if new_size.width != old_size.0 || new_size.height != old_size.1 {
+            if (new_size.width - old_size.0).abs() > FLOATING_PT_ERROR ||
+               (new_size.height - old_size.1).abs() > FLOATING_PT_ERROR {
```

Comment:
see above, there's no reason to not check exact values here
 - it's checking cached values, not computed values
 - it won't matter if it's a false positive
 - it casts to int right after (if anything it could cast to int first, but there is no need to do that either)